python/ray/serve/utils.py

```diff
@@ -85,23 +82,19 @@ def data(self):
 
 
 def parse_request_item(request_item):
```

Comment:
I think this function is now small enough to be moved into just the caller in backend_worker

Reply:
It turns out it's called from two different places (invoke_single and invoke_batch) so I opted to keep it as its own function, but let me know if it still makes sense to do it the other way.